provider/SqlServer/DbTypeMapping.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) 2021 Jeevan James
+// This file is licensed to you under the MIT License.
+// See the LICENSE file in the project root for more information.
+using System.Data;
+
+namespace Datask.Providers.SqlServer;
+
+public static class DbTypeMapping
+{
+    public static SqlDbType GetSqlDbTypeMapping(string dbType)
```

Comment:
This method seems to be used just to check if the type is a timestamp. It is also specific to the SQL Server, which means that we lose the advantages of having providers.

Instead, we should add a new property to ColumnDefinition (`IsAutoGenerated`). Each provider will take care of populating this value.

Reply:
Used DbCommand with DbType and removed SqlDbType.